.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
Need to make sure to exclude babelrc from the npm package as it's causing issues with the React Native packager https://github.com/cyclejs/cyclejs/pull/495